communication.tex

```diff
@@ -1,20 +1,20 @@
-% Responsible: Amanda Bauer
+\section{Community Engagement}
 
-\section{Communication}
+Rubin Observatory will work closely with the community on the detailed design of the \esp. 
 
-Rubin Observatry 
+\subsection{Survey Cadence Optimization Committee}
+The \href{https://www.lsst.org/content/charge-survey-cadence-optimization-committee-scoc}{Survey Cadence Optimization Committee (SCOC)} is an advisory committee to the Rubin Observatory Operations Director consisting of 10 members from drawn almost entirely from the science community.
```

Comment:
consisting of 10 members from drawn almost -> consisting of 10 members drawn from almost